README.md

```diff
@@ -12,7 +12,11 @@ A todo-list project built with Webpack and ES6 modules
 
 ## Getting Started
 
-For this project we implemented adding and removing of books using JavaScript
+In this project I built a simple HTML list of To Do tasks using webpack and served by a webpack dev server along with ES6 modules.
+
+- The webpage will allow users to add lists easily using the input field.
+- The completed tasks will each have a line through, implying the task is complete.
+- The completed tasks can be removed all together using a button.
 
 ### Setup
 
```

Comment:
[OPTIONAL CHANGE] 
This change won't stop your project from being approved but will help you document your project well.

You could add these steps below the clone command
- Run `git clone https://github.com/yonas44/Todo-list_webpack.git`
- Run `cd Todo-list_webpack.git`
- Run `npm install`
-  Run `npm start`

These will help anyone who wants to get your project running on their computer have access easily.

Reply:
Okay thanks for the feedback, I will add them. Thanks!🙏